CONTRIBUTING.md

```diff
@@ -293,6 +293,7 @@ accepted types are:
   considered part of a refactor, build process updates, dependency bumps, or
   auxiliary tools and libraries updates (LuaRocks, Travis-ci, etc...).
 
+[Back to TOC](#table-of-contents)
```

Comment:
Note that "Back to TOC" links were not added in this section _on purpose_ (within `<h5>` tags, that is).